erpnext/patches/v14_0/set_salary_slip_loan_fields.py

```diff
@@ -0,0 +1,15 @@
+import frappe
+
+def execute():
+	frappe.reload_doc("HR", "doctype", "salary_slip_loan")
+
+	frappe.db.sql("""
+		UPDATE `tabSalary Slip Loan` cssl
+		INNER JOIN `tabRepayment Schedule` rps ON rps.name = cssl.loan_repayment_detail
+		INNER JOIN `tabLoan` loan ON loan.name = rps.parent
+		SET cssl.loan_type = loan.loan_type,
+			cssl.disbursement_date = loan.disbursement_date,
+			cssl.total_loan_amount = loan.total_payment,
+			cssl.total_amount_paid = loan.total_payment - rps.balance_loan_amount,
```

Comment:
change total_amount_paid to pending_loan_amount using Edit Doctype.

in patch, remove total_amount_paid line and add
cssl.pending_loan_amount = cssl.total_payment + rps.balance_loan_amount